coops-ui/src/views/StandaloneDirectorsFiling.vue

```diff
@@ -155,6 +157,7 @@ export default {
       earliestDateToSet: 'your last filing',
       isCertified: false,
       certifiedBy: '',
+      certifyFormValid: null,
```

Comment:
Should be false?

Reply:
I was going with what AnnualReport does with it, but yes it could totally be false instead.

Reply:
I'll change it and recommit.